vuepress/docs/next/tutorials/devops/manage-nginx.md

```diff
@@ -181,5 +186,5 @@ spec:
 ​
 ```
 ### References
-- [Installation](https://cert-manager.io/docs/installation/)
-- [Supported releases](https://cert-manager.io/docs/installation/supported-releases/)
\ No newline at end of file
+- [cert-manager Installation](https://cert-manager.io/docs/installation/)
```

Comment:
Minor. Capitalization seems odd since cert-manager isn't formatted. Maybe take cert-manager out of the bullets and change the header from References to 'References for cert-manager'?  Just an idea.